packages/DateRangePicker/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@paprika/date-range-picker",
+  "version": "0.1.0",
+  "description": "DateRangePicker component allows user to view and specify a date range through an entry fields or a calendar widgets.",
+  "main": "lib/index.js",
+  "author": "@paprika",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika.git",
+    "directory": "packages/DateRangePicker"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@paprika/calendar": "^0.1.1",
+    "@paprika/date-input": "^0.1.0",
+    "@paprika/helpers": "^0.2.10",
+    "@paprika/icon": "^0.2.15",
+    "@paprika/popover": "^0.3.18",
+    "@paprika/stylers": "^0.2.6",
+    "@paprika/tokens": "^0.1.13",
+    "prop-types": "^15.7.2"
+  },
+  "peerDependencies": {
+    "@paprika/l10n": "^0.4.2",
```

Comment:
you are missing core-js dependency, while will work without it because most of our apps already installed, it should be a peer-dependency, https://github.com/acl-services/paprika/blob/master/packages/Button/package.json#L17